.github/CODEOWNERS

```diff
@@ -32,6 +32,7 @@
 /dataflow/**/*                         @davidcavazos @GoogleCloudPlatform/python-samples-reviewers
 /datastore/**/*                        @GoogleCloudPlatform/cloud-native-db-dpes @GoogleCloudPlatform/python-samples-reviewers
 /dns/**/*                              @GoogleCloudPlatform/python-samples-reviewers
+/documentai/**/*                       @GoogleCloudPlatform/python-samples-reviewers
```

Comment:
Are there any teams that can own this?

Reply:
I mentioned this in my friction log. Balaji's team does not have any teamsync set up; this would be ideal. Ill see if I can take this on right now.